recipes/OpenPatchMiner/LICENSE

```diff
@@ -0,0 +1,9 @@
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```

Comment:
Where did this license file come from? It doesn't match [what upstream has](https://github.com/CBICA/OPM/blob/master/LICENSE.txt)